src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -1068,5 +1069,18 @@ public sealed override bool Equals(Symbol other, TypeCompareKind compareKind)
                 nps.Equals(this, compareKind) :
                 base.Equals(other, compareKind);
         }
+
+#nullable enable
+        internal bool DeriveCompilerFeatureRequiredUseSiteInfo(ref UseSiteInfo<AssemblySymbol> result, MetadataDecoder decoder)
+        {
+            PEUtilities.DeriveUseSiteInfoFromCompilerFeatureRequiredAttributes(
+                ref result,
+                this,
+                (PEModuleSymbol)ContainingModule,
+                Handle,
+                allowedFeatures: CompilerFeatureRequiredFeatures.None,
+                decoder);
+            return result.DiagnosticInfo != null && IsHighestPriorityUseSiteError(result.DiagnosticInfo);
```

Comment:
>IsHighestPriorityUseSiteError [](http://example.com/codeflow?start=52&length=29)

Is it valid to call this method for parameter? Parameters do not have use-site errors on their own.